### PR TITLE
Create merge request

### PR DESCRIPTION
**Pull request by NataliaHlazova**
Create merge request and submit author as reviewer;

### CODE REVIEW

File: README.ml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# cstd-hlazovanm-2122
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.ml:1]: Add task details and details about yourself

Comment (NataliaHlazova) [reply README.ml]: README.md has been updated.

Comment (andrii-s-shpitser-lpnu-ua) [reply README.ml]: Also add your details number, hw i-face and dd type

Comment (NataliaHlazova) [reply README.ml]: README.md has been updated#2

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**